lib/config/database.go

```diff
@@ -55,11 +55,16 @@ teleport:
   {{- end }}
 db_service:
   enabled: "yes"
-  # Matchers for database resources created with "tctl create" command.
-  # For more information: https://goteleport.com/docs/database-access/guides/dynamic-registration/
+  # Matchers for database resources created with "tctl create" command or by the discovery service.
```

Comment:
Should we add a conditional on rendering this section (including this comment and the `resources:` line)? This would make it consistent with other sections.

Reply:
I like the implemented behaviour where we leave an empty section with some comments and a link to a guide.

I think we should do that to other sections too. For example, if user only specifies a static database, it would be cool we leave some comments and links how to set up auto-discovery in this sample config.

Reply:
Consistency is important. I will make a separate change to update other sections.